auth/src/api/services/key-service.ts

```diff
@@ -1,32 +1,40 @@
-import { Keys } from '../../config';
+import { Config } from '../../config';
 import { KeyHandler } from '../interfaces/key-handler';
 
+const AUTH_DEV_KEY = 'auth-dev-key';
+
 export class KeyService extends KeyHandler {
-    protected privateTokenKey: string;
-    protected privateCookieKey: string;
-    protected publicTokenKey: string;
-    protected publicCookieKey: string;
+    protected tokenKey = AUTH_DEV_KEY;
```

Comment:
Do net et dev keys as default - it could be forgotten to set real ones in production. Better enforce them to be set -> raise an error if the env variables do not exist or are empty.